quickwit-metastore/src/metastore/single_file_metastore.rs

```diff
@@ -394,12 +421,14 @@ mod tests {
     use std::sync::Arc;
     use std::time::Duration;
 
-    use crate::{IndexMetadata, MetastoreError};
-    use crate::{Metastore, SingleFileMetastore, SplitMetadata, SplitState};
     use chrono::Utc;
+    use tokio::time::sleep;
+
     use quickwit_index_config::AllFlattenIndexConfig;
     use quickwit_storage::{MockStorage, StorageErrorKind};
-    use tokio::time::sleep;
+
+    use crate::{IndexMetadata, MetastoreError};
+    use crate::{Metastore, SingleFileMetastore, SplitMetadata, SplitState};
```

Comment:
Just curious, do you use an automated tool to rearrange the imports?

Reply:
Basically, I used the cargo fmt. After that, I sorted them by standard packages, third party packages, Quickwit packages, and the packages itself.
If you prefer to just do `cargo fmt`, I will do that.

Reply:
No, I like this ordering. We can't just expect people to follow the same convention without an automated way of doing so.